### DISCUSSION TITLE
مشخص کردن سایر روزهای تعطیل رسمی در تقویم به رنگ قرمز مانند جمعه ها.

### DISCUSSION THREAD

**Discussion by loveforall1** [majidh1/JalaliDatePicker#95] [Ideas]
مشخص کردن سایر روزهای تعطیل رسمی در تقویم به رنگ قرمز مانند جمعه ها.

**Comment by majidh1** [comment]
به وسیله dayRendering میتونید هم کلاس بدین به روزها هم تعطیلات رو مشخص کنید
https://codepen.io/majidh1/pen/ZEMPxJY
